src/utils/adjust-markup.js

```diff
@@ -168,11 +205,35 @@ async function adjustMDAST(context, action) {
     .filter(([_, cfg]) => cfg.type === 'markdown')
     .forEach(([name, cfg]) => {
       logger.info(`Applying markdown markup adjustment: ${name}`);
-      transformer.match(cfg.match, (h, node) => {
-        const handler = transformer.constructor.default(node);
+      transformer.match(cfg.match, function myhandler(h, node, parent) {
+        const [fallbackhandler] = transformer
+          .allmatches(node)
+          .filter((theirhandler) => theirhandler !== myhandler);
         // Generate the matching VDOM element
-        const el = handler(h, node);
-        return patchVDOMNode(el, cfg);
+
+        /**
+     * A function that enables the recursive processing of MDAST child nodes
+     * in handler functions.
+     * @param {function} callback the HAST-constructing callback function
+     * @param {Node} childnode the MDAST child node that should be handled
+     * @param {Node} mdastparent the MDAST parent node, usually the current MDAST node
+     * processed by the handler function
+     * @param {*} hastparent the HAST parent node that the transformed child will be appended to
+     */
+        function handlechild(callback, childnode, mdastparent, hastparent) {
```

Comment:
This has the biggest potential impact (because it recurses), but it's part of the bugfix and the trees are relatively small.